iOSNativeTemplate/install.js

```diff
@@ -1,7 +1,20 @@
 #!/usr/bin/env node
 
+var packageJson = require('./package.json')
 var execSync = require('child_process').execSync;
-console.log('Installing npm dependencies');
-execSync('npm install', {stdio:[0,1,2]});
+var path = require('path');
+var fs = require('fs');
+
+console.log('Installing sdk dependencies');
+for (var sdkDependency in packageJson.sdkDependencies) {
+    var repoUrlWithBranch = packageJson.sdkDependencies[sdkDependency];
+    var parts = repoUrlWithBranch.split('#'), repoUrl = parts[0], branch = parts.length > 1 ? parts[1] : 'master';
```

Comment:
I read the SDK repo URL from the package.json. Then I do a git clone (shallow). But first I need to extract the tag or branch from the URL. Repo#branch is a syntax understood by npm not git. If the URL doesn't have a tag or branch specified then it means one wants master. I could also drop the branch option from the git clone command in that case.